Audacia.Quest/Audacia.Quest/Pages/Index.razor

```diff
@@ -1,70 +1,127 @@
 ﻿@page "/"
 
 @using System.Drawing
+@using Audacia.Quest.Core
+@using Audacia.Quest.Core.Events
+@using Audacia.Quest.Core.Params
 @using Blazor.Extensions
 @using Blazor.Extensions.Canvas.Canvas2D
 
 @inject IJSRuntime JsRuntime
 
-<div id="canvasContainer" style="position: fixed; opacity: 1; background-color: #6495ED; width: 100%; height: 100%">
+@*<img @ref="_fox" hidden src="assets/fox.png" />*@
+@if (Initialized)
+{
+    <Assets Context="_gameContext" />
+}
+
+<div id="canvasContainer" style="background-color: #6495ED; position: fixed; opacity: 1; width: 100%; height: 100%">
```

Comment:
It's good practice to use classes rather than inline styles, I'd recommend looking into css isolation.
https://docs.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-6.0
This can be easily done using a `Index.razor.css` file.